types/evidence.go

```diff
@@ -219,13 +219,13 @@ func NewMockRandomGoodEvidence(height int64, address []byte, randBytes []byte) M
 }
 
 func (e MockRandomGoodEvidence) Hash() []byte {
-	return []byte(fmt.Sprintf("%d-%x", e.Height_, e.randBytes))
+	return []byte(fmt.Sprintf("%d-%x", e.EvidenceHeight, e.randBytes))
 }
 
 // UNSTABLE
 type MockGoodEvidence struct {
-	Height_  int64
-	Address_ []byte
+	EvidenceHeight  int64
+	EvidenceAddress []byte
```

Comment:
```suggestion
	Address []byte
```

?

Reply:
Same as my above comment on the `Height` method.